source/frontend/src/features/mapSideBar/property/tabs/propertyDetailsManagement/detail/summary/PropertyManagementDetailContainer.tsx

```diff
@@ -0,0 +1,55 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { usePropertyManagementRepository } from '@/hooks/repositories/usePropertyManagementRepository';
+import { Api_Property, Api_PropertyManagement } from '@/models/api/Property';
+
+import { EditManagementState } from '../../../../PropertyViewSelector';
+import { IPropertyManagementDetailViewProps } from './PropertyManagementDetailView';
+
+interface IPropertyManagementDetailContainerProps {
+  property: Api_Property;
+  setEditManagementState: (state: EditManagementState | null) => void;
+  View: React.FC<IPropertyManagementDetailViewProps>;
+}
+
+export const PropertyManagementDetailContainer: React.FunctionComponent<
+  IPropertyManagementDetailContainerProps
+> = ({ property, setEditManagementState, View }) => {
+  const [propertyManagement, setPropertyManagement] = useState<Api_PropertyManagement>({
+    id: property.id ?? 0,
+    rowVersion: null,
+    managementPurposes: [],
+    additionalDetails: null,
+    isUtilitiesPayable: null,
+    isTaxesPayable: null,
+    isLeaseActive: false,
+    isLeaseExpired: false,
+    leaseExpiryDate: null,
+  });
+
+  const {
+    getPropertyManagement: { execute: getPropertyManagement, loading },
+  } = usePropertyManagementRepository();
+
+  const fetchPropertyManagement = useCallback(async () => {
+    if (!property.id) {
+      return;
+    }
+    const response = await getPropertyManagement(property.id);
+    if (response) {
+      setPropertyManagement(response);
```

Comment:
strictly speaking I think you should either be checking if the component is still mounted here, or skip using this state and use the data object that is returned from the repository.

Reply:
Will change - I was using the same pattern as Manuel on this property contact PR

Reply:
I will change this in my next PR